PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- Please verify that your code satisfies the
+[code/documentation quality guidelines](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines).
+- Please prefix the title of your pull request with `[MRG]` if the
+contribution is complete and should be subjected to a detailed review.
+Otherwise:
```

Comment:
You probably need to leave a new line before "otherwise". Look at how github renders the markdown [here](https://github.com/nelson-liu/scikit-learn/blob/create_issue_pr_template/PULL_REQUEST_TEMPLATE.md)